<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>396</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>453</width>
    <height>396</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>453</width>
    <height>396</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>453</width>
    <height>396</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>libMule Toolchain Configuration Editor</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>libMule Toolchain Configuration Editor version ^V^
Copyright (c) Tim K ^Y^. Licensed under ^L^.</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentConfigLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current configuration:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="currentConfigEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="currentConfigSelectButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="compilerCppLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C/C++ compiler path:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compilerCppSelectButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="compilerCppEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="compilerFlagsLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Compiler flags:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="compilerFlagsEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="linkerLdLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Linker path:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linkerLdEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="linkerLdSelectButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="linkerFlagsLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Linker flags:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linkerFlagsEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="deployLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote copy command:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deployEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="runLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote run command:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="runEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sdkPathLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>libMule SDK path:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sdkPathEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sdkPathSelectButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="targetLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>441</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current target: ^T^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>453</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_this_tool"/>
    <addaction name="actionAbout_Qt_framework"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout_this_tool">
   <property name="text">
    <string>About this tool</string>
   </property>
  </action>
  <action name="actionAbout_Qt_framework">
   <property name="text">
    <string>About Qt framework</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
